app/main.py

```diff
@@ -1,8 +1,34 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: Person) -> list:
+    person_instances = []
+
+    for person_data in people:
+        name = person_data["name"]
+        age = person_data["age"]
+
+        person = Person(name, age)
```

Comment:
you can create Person instances using list comprehension and then set the wife/husband attribute separately using another loop